improvelib/utils.py

```diff
@@ -587,36 +578,29 @@ def store_predictions_df(params: Dict,
 def compute_performance_scores(params: Dict,
                                y_true: np.array,
                                y_pred: np.array,
-                               stage: str,
-                               outdir: Union[Path, str],
-                               metrics: List):
+                               stage: str):
```

Comment:
Same here, helper functions should not use implicit knowledge of what params is.

Reply:
Another option is to pass in the config object.